src/app/[locale]/incident-map/components/IncidentMapContent.tsx

```
@@ -18,16 +18,21 @@ import {
   Icon,
   IconButton,
   MapMarker,
+  SpotlightSection,
 } from '@/components'
 import { useConfig } from '@/contexts/ConfigContext'
 import { Map } from '@/components/ui/Map'
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)